src/email-templates/rejection_notification.handlebars

```diff
@@ -1,8 +1,7 @@
 <p>Hello {{companyName}},</p>
 <br>  
-<p>We have reviewed your application for {{companyName}} and decided to reject your application.</p>    
-<p>Some common causes of rejection are spam, abuse or a motivation that does not align with our principles. 
```

Comment:
I don't think this should be removed, it provides more context as to why some applications might be removed in the future. Thoughts? @odracir04

Reply:
I agree. It does make it clearer to the applicant. Will place it back